### PR TITLE
Text input border with box-shadow combo

### PR DESCRIPTION
**Pull request by RVMendoza**
Fixes https://github.com/sparkdesignsystem/spark-design-system/issues/3894

Input borders are now a 1px border with an additive 1px box-shadow on hover or focus.

### CODE REVIEW

File: styles/tokens/input.json [hunk 1]
```diff
@@ -1981,9 +2209,24 @@
         "file": "settings",
         "themable": true
       },
-      "border-color-hover": {
-        "comment": "The border color of the custom checkbox input when hovered.",
-        "value": "{black-tint.70.value}",
+      "border-hover": {
+        "comment": "The border the custom checkbox input when hovered.",
+        "value": "1px solid {black.value}",
```
Comment (guern1kn) [inline styles/tokens/input.json:2214]: add attributes - content here

File: styles/base/inputs/_select.scss [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
   &:focus {
     border-color: $sprk-select-focus-border-color;
+    box-shadow: $sprk-select-focus-box-shadow;
```
Comment (afebbraro) [inline styles/base/inputs/_select.scss:23]: The selects in the deploy preview are missing the shadow: https://deploy-preview-3962--spark-sb-html.netlify.app/?path=/story/components-input-select--select-box

**Review by afebbraro — CHANGES_REQUESTED** [review]
Looks good so far besides for the few comments I left on here. Thanks!

File: html/base/inputs/Radio.stories.js [hunk 3]
```diff
@@ -701,7 +701,7 @@ export const hugeDisabled = () => {
   return `
     <div class="sprk-b-InputContainer sprk-b-InputContainer--huge">
       <fieldset class="sprk-b-Fieldset">
-        <legend class="sprk-b-Legend sprk-b-Legend--disabled">
```
Comment (bobwise) [inline html/base/inputs/Radio.stories.js:704]: `sprk-b-Legend--disabled` does not exist

**Review by guern1kn — APPROVED** [review]

**Review by afebbraro — APPROVED** [review]
thanks!

**Pull request merged**